src/api-explorer/v4-0/Request.swagger2.json

```diff
@@ -0,0 +1,3922 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Manage documents used for pre-spend authorizations within Concur Request.",
+    "version": "4.0",
+    "title": "Request"
+  },
+  "host": "www.concursolutions.com",
+  "basePath": "/api/v4",
+  "tags": [
+    {
+    "name": "Resources",
+    "description": " "
+    }
+  ],
+  "paths": {
+    "/travelrequest/requests": {
```

Comment:
I make the same path direction as the /api-exporer/v3-0/Request.swagger2.json. The api path is "www.concursolutions.com/api/**v4**/**travelrequest**/requests". The invoice documentation in v4-0 uses "www.concursolution.com/**invoice**/provider-payment/**v4**/payments" as path. Which URL is correct? The "v4" as top level resource or project as top level resource? Thanks for help.

Reply:
Check https://developer.concur.com/api-explorer/ API v3 is using the pre2017 authentication therefore the path was https://www.concursolutions.com/api/{version}
Now we have Oauth2 authentication, therefore the path should be https://us.api.concursolutions.com

Reply:
Since you will be using https://us.api.concursolutions.com the path to Request API v4 should be
https://us.api.concursolutions.com/travelrequest/v4 as written here https://developer.concur.com/api-reference/request/v4.examples.html#request-resource-create

Therefore I expect the path to be "/travelrequest/v4/requests"

I would be great if this could be confirmed by someone with experience on swagger doc from developer.concur.com, like @charlieconcur maybe

Reply:
I change the path to /travelrequest/v4/subpath